lib/chess/pieces/piece.rb

```diff
@@ -0,0 +1,116 @@
+require_relative '../chess'
+require_relative '../step'
+require 'colorize'
+require 'colorized_string'
+require 'byebug'
+
+module Chess
+  class Piece
+    attr_reader :symbol, :team, :current_step
```

Comment:
In ruby it's general practice that you follow the namespaces according to the folder organization. If this file is in `lib/chess/pieces/piece.rb` then this should be `Chess::Pieces::Piece`